src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -175,7 +175,7 @@ public static CliArguments parse(String[] args) throws HelpScreenException, Pars
             Date sinceDate = cliSinceDate.orElse(getDateMinusAMonth(cliUntilDate));
             Date untilDate = cliUntilDate.orElse(getCurrentDate());
             List<String> locations = results.get(REPO_FLAGS[0]);
-            List<Format> formats = Format.convertStringsToFormats(results.get(FORMAT_FLAGS[0]));
+            List<FileType> formats = FileType.convertFormatStringsToFileTypes(results.get(FORMAT_FLAGS[0]));
```

Comment:
Should rename `formats` too?

Reply:
Would prefer to retain `formats`. `fileTypes` may confuse others